apps/emqx_auth/src/emqx_authn/emqx_authn_api.erl

```diff
@@ -1253,6 +1292,19 @@ serialize_error({unknown_authn_type, Type}) ->
         code => <<"BAD_REQUEST">>,
         message => binfmt("Unknown type '~p'", [Type])
     }};
+serialize_error(#{not_found := NotFound, not_reordered := NotReordered}) ->
+    NotFoundFmt = "Authenticators: ~p are not found",
```

Comment:
will it produce `[<<"redis">>, <<"mysql">>]`?
if yes, maybe better to use `[~ts]` for a `lists:join(",", NotFound)`

Reply:
You are right, thanks!

Reply:
Done